static/templates/narrow_to_compose_recipients_tooltip.hbs

```diff
@@ -1,4 +1,7 @@
-<span>{{t 'Go to conversation' }}</span> <span class="hotkey-hint">({{shortcut_html}})</span>
-{{#if display_current_view}}
-<p class="narrow_to_compose_recipient_current_view_help">{{display_current_view}}</p>
-{{/if}}
+<div class="narrow_to_compose_recipients content">
```

Comment:
Is this CSS class used for something? I'm also note sure if the space before `content` is intentional. I could imagine this just being a `div` without being a label if you just need to define a block here.

Reply:
Fixed by removing the class attribute from the `div` node. The space before `content` was unintentional indeed, thanks for spotting it.